arcgis-ios-sdk-samples/Edit data/Edit features (connected)/EditFeaturesOnlineViewController.swift

```diff
@@ -139,7 +146,6 @@ class EditFeaturesOnlineViewController: UIViewController, AGSGeoViewTouchDelegat
         //disable the done button until any geometry changes
         self.doneBBI.isEnabled = false
         
-        NotificationCenter.default.addObserver(self, selector: #selector(EditFeaturesOnlineViewController.sketchChanged(_:)), name: .AGSSketchEditorGeometryDidChange, object: nil)
```

Comment:
It isn't immediately clear why this was where it was and what the consequences of moving it would be. I think a better option would be to leave this where it was (since moving it wasn't needed to satisfy the rule) and then update the removal to specify the the notification name, namely `AGSSketchEditorGeometryDidChange`.

Same with `OfflineEditingViewController.swift`.